bootcd/genesis.ks.template

```diff
@@ -220,6 +220,8 @@ echo ">>>>> Removing bloat from /usr/share"
 rm -rf /usr/share/doc
 rm -rf /usr/share/icons
 
+sed -e 's|ACTIVE_CONSOLES.*|ACTIVE_CONSOLES="/dev/tty1 /dev/ttyS0 /dev/ttyS1"|' -i /etc/sysconfig/init
```

Comment:
IIRC the content of that file on a freshly built Genesis image without my change was just `/dev/tty1`. I have no objection to updating this to look more like `/dev/tty[1-6] /dev/ttyS[0-1]` if that would be preferable to others and also works.